parquet/src/file/properties.rs

```diff
@@ -193,6 +193,13 @@ impl WriterProperties {
         WriterPropertiesBuilder::default()
     }
 
+    /// Converts this [`WriterProperties`] into a [`WriterPropertiesBuilder`]
+    /// Used for mutating existing property settings
+    pub fn into_builder(self) -> WriterPropertiesBuilder {
+        let builder: WriterPropertiesBuilder = self.into();
+        builder
```

Comment:
I think you could make this more concise like this if you want:

```suggestion
        self.into()
```